service/http.directory/src/test/java/org/kaazing/gateway/service/http/directory/HttpDirectoryServiceIT.java

```diff
@@ -179,11 +179,6 @@ public void testGetForbiddenFileExists() throws Exception {
     }
 
     // /////////////////// HOST HEADER ///////////////////////
-    @Specification("host/host.empty.header.with.absolute.uri")
```

Comment:
RFC definition of proxy is forwarding proxy. Our http.proxy needs to be interpreted as gateway or server.